addons/mrp/views/mrp_unbuild_views.xml

```diff
@@ -108,7 +108,7 @@
                     <sheet>
                         <div class="oe_button_box" name="button_box">
                             <button class="oe_stat_button" name="%(action_mrp_unbuild_moves)d"
-                                    string="Product Moves" type="action" icon="fa-arrows-v" states="done"/>
+                                    string="Product Moves" type="action" icon="fa-exchange" states="done"/>
```

Comment:
@svs-odoo it sends back to the stock.moves, I'd prefer to show the stock.move.lines. 
@sle-odoo What do you think of it ?

Reply:
@lap-odoo yes it would be more consistent with the other modules